### PR TITLE
Rename CTX_* constants by CONTEXT_*

### PR DESCRIPTION
**Pull request by JesusValera**
# Description

From my point of view, it is better to have long but pronounceable nouns than abbreviations. :zipper_mouth_face: 

# Changes

- Replace `CTX_STMT` by `CONTEXT_STATEMENT`
- Replace `CTX_EXP` by `CONTEXT_EXPRESSION`
- Replace `CTX_RET` by `CONTEXt_RETURN`

### CODE REVIEW

**Review by Chemaclass — APPROVED** [review]
Yes ⚡ !

File: src/php/NodeEnvironment.php [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 final class NodeEnvironment
 {
-    public const CTX_EXPR = 'expr';
-    public const CTX_STMT = 'stmt';
-    public const CTX_RET = 'ret';
+    public const CONTEXT_EXPRESSION = 'expr';
```
Comment (jenshaase) [inline src/php/NodeEnvironment.php:11]: You could also rename the strings to 'expression', 'statement' and 'return'. I don't think the strings are used somewhere other than with these constants.

**Review by jenshaase — CHANGES_REQUESTED** [review]

File: src/php/NodeEnvironment.php (hunk 1 above)
Comment (JesusValera) [reply src/php/NodeEnvironment.php]: Agreed! :)

**Review by jenshaase — APPROVED** [review]
Super!

**Pull request merged**